tests/manage/test_pvc_disruptive.py

```diff
@@ -46,7 +47,11 @@ def disruptive_base(self, operation_to_disrupt, resource_to_delete):
         assert self.pod_obj.ocp.wait_for_resource(
             condition=constants.STATUS_RUNNING, resource_name=self.pod_obj.name, timeout=120
         )
-        self.pod_obj.run_io(storage_type=self.storage_type, size='1G')
+        self.pod_obj.run_io(
+            storage_type=self.storage_type, size='1G', wait=False
+        )
+        # wait for atleast 10 seconds for IO to start
```

Comment:
maybe we can verify if the IO started from within run_io()?
For example, in ocs_ci.ocs.resources.pod.run_io_in_bg, we check for file existence after starting the IO of creating and writing to files - ocs_ci.ocs.resources.pod.check_file_existence

Reply:
@ebenahar the run_io_in_bg is different, and run_io uses fio, I think we are safe here with 10 seconds of wait and testing.

Reply:
Why not using self.fio_thread.running()?

Reply:
the thread will run in nano second, what I am waiting for here is for sometime so that I/O's are in flight. More sleep doesn't hurt here.

Reply:
I don't want to overcomplicate things but:
1) what will happen if yum install fio hangs?
2) why not to create a simple function that actually checks that the df is changing and the disk size is being consumed.

We have to be accurate and that's why sleep is bad usage - everywhere

Reply:
@RazTamir 

> what will happen if yum install fio hangs?

I assume `yum` itself would time out and we would end up raising a `CommandFailed` exception. I couldn't find where this install actually happens though so that assumption could be wrong. 

> why not to create a simple function that actually checks that the df is changing and the disk size is being consumed.

I think this is a good idea, we should be verifying actual disk activity.

> We have to be accurate and that's why sleep is bad usage - everywhere

Perhaps we could be more efficient and have the function that verifies disk activity return a new custom exception that our `retry` decorator expects and utilize that instead of just sleeping.

Reply:
This is not *my* test :) (the test itself is quite disruptive here compared to other disruptive tests), but as a part of signature change, I noticed that some sleep is better than nothing to give I/O some chance - we are using stable tools to test I/O so I am not super concerned about the tool failure and we have routines to check for stats).

we have method to get the FIO stats but because of the nature of *this* test, I cannot use it here, I will try to add something here to be sure, but sleep as such is not a bad thing when used appropriately, its been used across various projects(including rook ;-).

Reply:
> I don't want to overcomplicate things but:
> 
> 1. what will happen if yum install fio hangs?

one shouldn't run yum install within a container. That's not the way containers are designed. I see that we do that as part of our test today. I'm going to raise an issue to use a custom container image with fio pre-installed. 

> 2. why not to create a simple function that actually checks that the df is changing and the disk size is being consumed.
> 
> We have to be accurate and that's why sleep is bad usage - everywhere

Reply:
@RazTamir  Is this something we need to hold this PR up for or are we okay with moving forward with these changes and potentially addressing the concern around using sleep in a subsequent PR?

Reply:
We can handle it later but we'll need to track it we new issue

Reply:
I will file a ticket as follow on PR and merge this one after rebase and new testing.